config/fcos/v1_5_exp/validate.go

```diff
@@ -46,3 +47,10 @@ func (e Extension) Validate(c path.ContextPath) (r report.Report) {
 	}
 	return
 }
+
+func (user GrubUser) Validate(c path.ContextPath) (r report.Report) {
+	if util.NotEmpty(user.Key) && !util.NotEmpty(user.User) {
```

Comment:
Is there any reason to accept GrubUsers with missing usernames?

Reply:
You're right, there isn't. Will fix the condition :+1: